src/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -50,6 +51,7 @@
   <ItemGroup Condition="'$(TargetGroup)' == 'net45'">
     <TargetingPackReference Include="mscorlib" />
     <TargetingPackReference Include="System.Data" />
+    <Compile Include="System\Data\Common\DbDataReaderExtensions.Facade.cs" />
```

Comment:
From the naming convention guidelines (https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md#code-file-naming-conventions) this should be DbDataReaderExtensions.net45.cs.